signing-service-challenge/README.md

```diff
@@ -2,23 +2,23 @@
 
 ## Instructions
 
-This challenge is part of the SWE interview process at fiskaly.
+This challenge is part of the software engineering interview process at fiskaly.
 
 If you see this challenge, you've passed the first round of interviews and are now at the second and last stage.
 
-We would ask you to prepare the challenge below to have a basis for discussion in the skill fit interview with two colleagues from the development department.
+We would like you to complete the challenge below. The two colleagues from the development department will then discuss your solution with you in the skill fit interview.
 
-`quality > quantity`
+The `quality` of your code is more important to us than the `quantity`.
```

Comment:
If we put it like this I would ditch the back ticks...

Reply:
fixed